pkg/openfeature/provider.go

```diff
@@ -12,15 +12,19 @@ const (
 	TargetingKey string = "targetingKey" // evaluation context map key. The targeting key uniquely identifies the subject (end-user, or client service) of a flag evaluation.
 )
 
+// TransformedContext contains metadata for a given flag evaluation in a flattened structure.
+// TargetingKey ("targetingKey") is stored as a string value if provided in the evaluation context.
+type TransformedContext map[string]interface{}
```

Comment:
Nitpick:

transformedContext is used in a lot of SDKs as a variable name internally because it was the context provided after the contextTransformer function ran.

I'm fine with it here, but I wonder if FlattenedContext might be a bit more helpful.

Reply:
updated